addons/hr_holidays/report/hr_leave_reports.xml

```diff
@@ -71,7 +71,7 @@
                         <div t-attf-class="oe_kanban_global_click">
                             <div>
                                 <span>
-                                    <img t-att-src="kanban_image('hr.employee', 'image_64', record.employee_id.raw_value)" t-att-title="record.employee_id.value" t-att-alt="record.employee_id.value" width="40" height="40" class="oe_kanban_avatar float-left mr4"/>
+                                    <img t-att-src="kanban_image('hr.employee', 'image_128', record.employee_id.raw_value)" t-att-title="record.employee_id.value" t-att-alt="record.employee_id.value" class="oe_kanban_avatar o_image_40 float-left mr4"/>
```

Comment:
Is not kanban avatar forcing a width/height? So the class is useless ?

Reply:
`oe_kanban_avatar` is only forcing width/height if inside `.o_kanban_record_bottom .oe_kanban_bottom_right`, not the case here

Reply:
:nauseated_face: :mask:

Reply:
Is this considered resolved until improvement some day?